added paragraph explaining why Freshness section is so detailed (#372)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

and that any Attestation Result was generated using the latest Appraisal Policy
for Evidence.

> This section appears very detailed.

Reviewer: ```suggestion
This section provides a number of details.
```

It does not however define any protocol formats, the interactions shown are abstract.
This section is intended for those creating protocols and solutions to understand the options available to ensure freshness.
The way in which freshness is provisioned in a protocol is an architectural decision.

> Provisining of freshness has an impact on needed round trips in a protocol, and therefore must be made very early in design.

Reviewer: ```suggestion
Provisioning of freshness has an impact on the number of needed round trips in a protocol, and therefore must be made very early in design.
```

> Different decisions will have significant impacts on resulting interoperability.
> This concern is why this section goes into sufficient detail such that choices in freshness will be compatible across interacting protocols, such as depicted in {{multievidence_diag}}.

Reviewer: ```suggestion
Different decisions will have significant impacts on resulting interoperability,
which why this section goes into sufficient detail such that choices in freshness will be compatible across interacting protocols, such as depicted in {{multievidence_diag}}.
```

> which why this section goes into sufficient detail such that choices in freshness will be compatible across interacting protocols, such as depicted in {{multievidence_diag}}.

Author: ```suggestion
which is why this section goes into sufficient detail such that choices in freshness will be compatible across interacting protocols, such as depicted in {{multievidence_diag}}.
```
